1-WebApp-OIDC/1-1-MyOrg/Startup.cs

```diff
@@ -32,16 +34,19 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             // Sign-in users with the Microsoft identity platform
-            services.AddMicrosoftIdentityPlatformAuthentication(Configuration);
+            //services.AddSignIn(Configuration);
+            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
```

Comment:
In order to give the flexibility of having many authentication schemes configured, `AddSignIn` is built on top of an `AuthenticationBuilder` so developers can have many schemes with different names, without having the extension library  overwriting their configs or hardcoding a specific scheme name. 

The default is `OpenIdConnectDefaults.AuthenticationScheme` (OpenIdConnect).